jina/helper.py

```diff
@@ -919,6 +919,41 @@ def is_valid_local_config_source(path: str) -> bool:
         return False
 
 
+secrets_regex_str = r'\${{\s?SECRETS\.[a-zA-Z0-9_]*\s?}}|\${{\s?secrets\.[a-zA-Z0-9_]*\s?}}'
+secret_name_regex_str = r'\.(.*?)(\s|\})'
+secrets_regex = re.compile(
+    secrets_regex_str
+)  # matches expressions of form '${{ SECRETS.var }}' or '${{ secrets.var }}'
```

Comment:
If I am not wrong it also matches `'${{SECRETS.var}}'` or `'${{secrets.var}}'` (because of `\s?`). I think in the JAML parser we enforce the space, should we do the same here?

Reply:
I do not really see why